docs/reference/esql/functions/greatest.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[esql-greatest]]
+=== `GREATEST`
+
+Returns the maximum value from many columns.
+
+[source.merge.styled,esql]
+----
+include::{esql-specs}/math.csv-spec[tag=greatest]
+----
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+include::{esql-specs}/math.csv-spec[tag=greatest-result]
+|===
+
+NOTE: When run on `keyword` or `text` fields, this'll return the last string
```

Comment:
Is this about multi-values support?

Reply:
They are like most of our other functions at this point and don't support multivalued fields. This is a case where it'd make a lot of sense to take, like, `mv_min` for `least`.

Reply:
It's about what `LEAST` and `GREATEST` mean when you use them with string style fields. Let me see about clarifying.